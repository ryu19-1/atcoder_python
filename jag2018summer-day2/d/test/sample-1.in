10
7
281614559 249378726 433981056 466775634 683612866
727071329 787572584 591471796 328464426 757737734
279580343 240336097 538846427 808491898 224313807
222498984 42804452 371605808 667115067 791865961
68683864 1045549765 515479514 1067782238 349547144
907343711 381772625 149003422 879314974 953881571
883899098 700164610 414212891 752949213 972845634
