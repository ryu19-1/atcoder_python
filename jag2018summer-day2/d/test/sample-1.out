10
0
-1
21
-1
11
111
